apps/settings/js/panels/date_time/panel.js

```diff
@@ -23,6 +23,9 @@ define(function(require) {
           clockTime: panel.querySelector('.clock-time'),
           timeManual: panel.querySelector('.time-manual'),
           timezone: panel.querySelector('.timezone'),
+          timezoneSelectors: [].slice.apply(
```

Comment:
For me, `selector` means the string to match related elements, do you think we should use a better naming like `timezoneElements` ... etc ?